content/english/blog/dotnet_are_we_webassembly_yet.md

```diff
@@ -0,0 +1,115 @@
+---
+title : ".NET: Are we WebAssembly Yet?"
+image : "images/blogs/dotnet_wasm.png"
+date: 2022-11-12T9:00:00-04:00
```

Comment:
Will need to change this for the current date but that can wait until an approve or two